packages/mobile/src/app/reducers.ts

```diff
@@ -39,11 +39,15 @@ export const appReducer = (
       return {
         ...state,
         ...getRehydratePayload(action, 'app'),
-        doingPinVerification: initialState.doingPinVerification,
```

Comment:
Should we be reseting `requestingAndroidPermission` here like we did for doingPinPerm?

Reply:
👍